.github/workflows/build.yml

```diff
@@ -2,7 +2,9 @@ name: Prime Data Hub
 
 on:
   pull_request:
-    branches: [ master, production ]
+    # Disable master branch for pentesting
+    # branches: [ master, production ]
+    branches: [ master ]
```

Comment:
Shouldn't this remove `master` not `production`?

Reply:
Oops...that was a bad copy/paste.  This is fixed.